hazelcast/src/main/java/com/hazelcast/util/scheduler/ScheduleType.java

```diff
@@ -17,5 +17,16 @@
 package com.hazelcast.util.scheduler;
 
 public enum ScheduleType {
```

Comment:
Could you add a line of comment for the meaning of ScheduleType? Because it is unclear to me how this fits in, and therefor the enum values don't make a lot of sense.